server/src/seeds/testdata.ts

```diff
@@ -171,35 +71,302 @@ const createTestUsers = async () => {
+
+  const customer3 = await Customer.query().insert({
+    roadmapId: roadmaps[0].id,
+    name: 'Customer 3',
+    color: '#FF5C4D',
+    email: 'customer3@webuystuff.com',
+  });
+
+  const customer4 = await Customer.query().insert({
+    roadmapId: roadmaps[2].id,
+    name: 'Customer 4',
+    color: '#4D99FF',
+    email: 'customer3@webuystuff.com',
+  });
+
+  const adminUser = await getUser('AdminPerson1');
+  await adminUser
+    .$relatedQuery('representativeFor')
+    .relate([customer1, customer2, customer3, customer4]);
+};
+
+const testRatings = async () => {
+  const adminId = (await getUser('AdminPerson1')).id;
+  const developer1Id = (await getUser('DeveloperPerson1')).id;
+  const developer2Id = (await getUser('DeveloperPerson2')).id;
+  const business1Id = (await getUser('BusinessPerson1')).id;
+  const business2Id = (await getUser('BusinessPerson2')).id;
+  const customerId = (await Customer.query().findOne('name', 'Customer 1')).id;
+  const customer2Id = (await Customer.query().findOne('name', 'Customer 2')).id;
+  const customer3Id = (await Customer.query().findOne('name', 'Customer 3')).id;
+
+  return [
+    {
+      createdBy: { id: developer1Id },
+      dimension: TaskRatingDimension.RequiredWork,
+      value: 3,
+      comment: 'No big deal',
+    },
+    {
+      createdBy: { id: developer2Id },
+      dimension: TaskRatingDimension.RequiredWork,
+      value: 7,
+      comment: 'I would argue this requires more work',
+    },
+    {
+      createdBy: { id: adminId },
+      createdFor: { id: customerId },
+      dimension: TaskRatingDimension.BusinessValue,
+      value: 7,
+    },
+    {
+      createdBy: { id: business1Id },
+      createdFor: { id: customerId },
+      dimension: TaskRatingDimension.BusinessValue,
+      value: 6,
+    },
+    {
+      createdBy: { id: adminId },
+      createdFor: {
+        id: customer2Id,
+      },
+      dimension: TaskRatingDimension.BusinessValue,
+      value: 4,
+    },
+    {
+      createdBy: { id: business2Id },
+      createdFor: { id: customer2Id },
+      dimension: TaskRatingDimension.BusinessValue,
+      value: 5,
+    },
+    {
+      createdBy: { id: adminId },
+      createdFor: { id: customer3Id },
+      dimension: TaskRatingDimension.BusinessValue,
+      value: 2,
+      comment: 'This is not relevant in their situation',
+    },
+  ];
+};
+
+const createTestTasks = async () => {
+  const defaultRatings = await testRatings();
+  const adminId = (await getUser('AdminPerson1')).id;
+  const createdBy = { id: adminId };
+
+  const tasks = [
+    {
+      name: 'Test task 1',
+      description: 'Test desc 1',
+      createdBy,
+      ratings: defaultRatings,
+    },
+    {
+      name: 'Test task 2',
+      description: 'Test desc 2',
+      createdBy,
+      ratings: defaultRatings.map(({ createdBy, createdFor, dimension }) =>
+        randomTaskratingValue(createdBy, createdFor, dimension),
+      ),
+    },
```

Comment:
Seems really good 👍 Good catches on the bugs! I think marking a few tasks completed (for example task 2 and 4) would be a nice finishing touch and then this is ready to be merged in my opinion.

```
    {
      name: 'Test task 2',
      description: 'Test desc 2',
      createdBy,
      ratings: defaultRatings.map(({ createdBy, createdFor, dimension }) =>
        randomTaskratingValue(createdBy, createdFor, dimension),
      ),
      completed: true
    },
```